### PR TITLE
Migrated succeed custom reports to B3

### PR DESCRIPTION
**Pull request by kkrampa**
@biyeun 

I migrated these reports to B3 but there is still  a problem with loader so it's not ready to merge until this problem is fixed.

### CODE REVIEW

**Comment by czue** [comment]
(added "don't merge" label)

File: custom/succeed/templates/succeed/ucla_table.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
-{% extends "reports/async/bootstrap2/tabular.html" %}
+{% extends "reports/async/bootstrap3/tabular.html" %}
 {% load hq_shared_tags %}
+
 {% block js-inline %}
     <script type="text/javascript">
```
Comment (biyeun) [inline custom/succeed/templates/succeed/ucla_table.html]: @kkrampa It looks like `{{block.super}}` is (on purpose?) not being invoked. To fix the loader, I recommend adding the following to `js-inline`
```
    <script type="text/template" id="js-template-loading-report">
        <div class="report-loading-container">
            <div class="report-loading">
                <h4>{% trans "Loading Report" %}</h4>
                <i class="fa fa-spin fa-spinner"></i>
            </div>
        </div>
    </script>
```

**Comment by biyeun** [comment]
@kkrampa looks good so far. made a suggestion for how to fix the loader

**Comment by biyeun** [comment]
@kkrampa since the loader was fixed in an earlier PR, merging this.

**Comment by biyeun** [comment]
@kkrampa actually can you do a last minute verification and sign-off that everything loads? I just want to be extra cautious because of this particular project. I will sign off tomorrow when I'm online immediately after deploy.

**Comment by kkrampa** [comment]
@biyeun I'm not able to do verification today. I will check this tomorrow.

File: corehq/apps/style/static/style/less/bootstrap3/report.less [hunk 2]
```diff
@@ -34,5 +34,5 @@
     height: 100%;
     background-color: @gray;
     // Fade for backdrop
-    .opacity(80);
+    .opacity(0.8);
```
Comment (kkrampa) [inline corehq/apps/style/static/style/less/bootstrap3/report.less]: Value of opacity is not divided by 100 in .opacity mixin in b3: http://getbootstrap.com/css/#less-mixins-opacity

**Comment by kkrampa** [comment]
Looks good on my local machine. I made one small tweak.

**Comment by biyeun** [comment]
@kkrampa ok sounds good. will be checking after deploy

**Pull request merged**